frontend/server.js

```diff
@@ -2,13 +2,17 @@ const webpackDevServer = require('webpack-dev-server');
 const webpack = require('webpack');
 
 const config = require('./webpack.config.js');
+const publicPath = 'https://958ba9a1711d.ngrok.io/dist/'
```

Comment:
Is this a placeholder for ngrok url?

Reply:
Hmm, I think I just left that in there and it should use the configured value. I'll remove it. Good catch!

Reply:
I'm trying to figure out why I had this in there in the first place. There were some issues getting this to work with ngrok. 

This configuration and frontend server is only used in local development, it should be compiled down to one server (just the backend) in production.

Reply:
I think I remember why I had this now. The backend code actually uses some of these variables from the frontend to figure out how to serve up the content. I'd temporarily hardcoded this but figured it out more and fixed it.